chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# This does what is necessary directly before the start of OS upgrade
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+mkdir -p "`dirname "$LOGFILE"`"
+
+# Copy stdout to fd 3
+exec 3>&1
+# Create fd 4 for logfile
+exec 4>> "$LOGFILE"
+# Set fd 1 and 2 to logfile
+exec 1>&4 2>&1
```

Comment:
> exec >>"$LOGFILE" 2>&1

I think this was just copy and pasted from install-chef-suse, where we still cared about stdout on the command line. and as i understand this script is just called from crowbar, and in no case by the user?